app/models/docker_parameter.rb

```diff
@@ -0,0 +1,21 @@
+class DockerParameter < ActiveRecord::Base
+  extend FriendlyId
+  friendly_id :key
+  include Parameterizable::ByIdName
+
+  validates_lengths_from_database
+
+  include Authorizable
+  validates :key, :presence => true, :no_whitespace => true
+
+  scoped_search :on => :key, :complete_value => true
+
+  default_scope -> { order("docker_parameters.key") }
+
+  before_validation :strip_whitespaces
```

Comment:
I mean, why not adding `validates :value, :presence => true,  :no_whitespace => true` too? Isn't it the same but reusing the code in the `no_whitespace` validation instead of writing your own `strip_whitespaces`?